docker/README.md

```
@@ -0,0 +1,142 @@
+
+If you encounter permission issues with volume mounts:
+
+```bash
+# Ensure proper ownership of plugin directories
+sudo chown -R 1001:1001 ~/.pulumicost
+```
+
+### Plugin Installation
+
+Currently, plugins must be manually installed in the `~/.pulumicost/plugins`
+directory. Future versions will include automated plugin downloading capabilities.
```

markdownlint-cli2 (0.18.1)

Files should end with a single newline character

(MD047, single-trailing-newline)